scripts/generators/tsconfig.js

```diff
@@ -39,7 +55,7 @@ function getTsPkgs(subRoot) {
         name === "@babel/compat-data" ||
         fs.existsSync(new URL(relative + "/src/index.ts", rootURL));
       if (!ret) {
-        console.log(`Skipping ${name} for tsconfig.json`);
+        // console.log(`Skipping ${name} for tsconfig.json`);
```

Comment:
Yeah, it was just annoying whenever we have empty folders due to changing branch :sweat_smile: 

I left it commented out so that we can re-enable it if we have to debug the script.